### PR TITLE
[8.0] Workflow tests need pilot.cfg to work

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

*test
FIX: Workflow tests need pilot.cfg to work

ENDRELEASENOTES

### CODE REVIEW

**Comment by chaen** [comment]
I think it should not be the default. The pilot can call it, specifying the `pilot.cfg` location

**Comment by fstagni** [comment]
It's because of https://github.com/DIRACGrid/Pilot/issues/188. Basically, all DIRAC commands would work also in a pilot environment.

File: tests/Workflow/Integration/Test_UserJobs.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 import DIRAC
 
-DIRAC.initialize()  # Initialize configuration
+DIRAC.initialize(extra_config_files=["pilot.cfg"])  # Initialize configuration
```
Comment (chrisburr) [inline tests/Workflow/Integration/Test_UserJobs.py:13]: This relies on people not changing the current working directory which I don't think is a reasonable assumption.

Comment (fstagni) [reply tests/Workflow/Integration/Test_UserJobs.py:13]: This is only run in Jenkins, where we know where we are.

**Review by atsareg — APPROVED** [review]

**Pull request merged**